aws-opentelemetry-distro/tests/amazon/opentelemetry/distro/test_instrumentation_patch.py

```diff
@@ -133,6 +178,56 @@ def _test_botocore_installed_flag(self):
             apply_instrumentation_patches()
             mock_apply_patches.assert_called()
 
+    def _test_patched_bedrock_instrumentation(self):
+        """For bedrock service, only on_success provides attributes, and we only expect to see guardrail"""
+        bedrock_sucess_attributes: Dict[str, str] = _do_bedrock_on_success()
```

Comment:
Use `_do_on_success_bedrock`, you need to show that everything other than `guardrailId` is ignored.

Reply:
Uplated